pkg/apis/eventing/v1alpha1/trigger_lifecycle.go

```diff
@@ -34,6 +34,8 @@ const (
 
 	TriggerConditionDependency apis.ConditionType = "Dependency"
 
+	TriggerConditionSubscriberReady apis.ConditionType = "SubscriberReady"
```

Comment:
I'm not a big fan of this name. As we aren't checking if the Subscriber is ready. We are checking if we can get its address. 

How about `SubscriberResolved`?

Reply:
SubscriberResolved was another suggestion I had, I think that's probably more appropriate...

Reply:
Updated the naming to `SubscriberResolved`